### ISSUE TITLE
Option to hide side panel

### ISSUE THREAD

**Issue by mooneyman52** [barryhunter/classygmaps#30]
I would love to hide the side panel so as to give the whole screen over to the map.

**Comment by pasiu-pl** [comment]
Me too!
Generally, Classymaps - it's a good job!!!
Big thanks&regards from Poland!!!

**Comment by MHVNY** [comment]
By all means.. a way to hide the side panel

**Comment by egoldobin** [comment]
i would say to make the separator between the side panel and the map movable. i have nothing against the side panel (a la classic maps), but sometimes it looks too narrow, esp. when it contains driving directions. I vote for movable separator.

**Comment by DenverDave** [comment]
Yep, this is the missing element that I first noticed.  Hide or allow to be collapsed to zero like the old Google Maps classic used to do.   I'd pay $1 to contribute if this would help.

**Comment by halitus** [comment]
I also vote for movable separator and hider button.

**Comment by chwaqasafzal** [comment]
Please allow users to adjust the width of the sidebar

**Comment by MHVNY** [comment]
and/or hide it all together..The main complaint of the the 'new'. google maps is clutter.. obstructing the view.., it is a 'map' not a freakn' a social page. Which seems to be the direction Google is going 
" I just saw a dog take a shit on Rt 17"
